.github/workflows/deploy_android_production.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy Android production build with APK file
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
```

Comment:
we only deploy the production build when merging a branch to `main`. Let's remove `'release/**'`